timeline-chart/src/components/time-graph-annotation.ts

```diff
@@ -11,8 +11,8 @@ export interface TimeGraphAnnotationStyle extends TimeGraphComponentOptions {
     symbol?: string
     size?: number
     color?: number
+    opacity?: number
     verticalAlign?: string
-    opacity?:number
```

Comment:
No point in this, right?

Reply:
There was missing space so I took the opportunity to move opacity next to color at the same time, for clarity ;)